.github/workflows/todo.yml

```diff
@@ -0,0 +1,17 @@
+name: TODO Bot
+on:
+  push:
+    branches:
+    - main
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@master
+    - name: Run tdg-github-action
+      uses: ribtoks/tdg-github-action@master
+      with:
+        TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
This is a pretty big attack vector isn't it?

Reply:
As per offline discussion decided to use a specific commit... for `checkout` we're using v4 anyways in other workflows.